sovrin_node/server/config_req_handler.py

```diff
@@ -63,6 +63,12 @@ def validate(self, req: Request, config=None):
             # present
             status = self.upgrader.statusInLedger(req.operation.get(NAME),
                                                   req.operation.get(VERSION))
+            if status == START and action == START:
+                raise UnauthorizedClientRequest(
```

Comment:
Why do we raise `UnauthorizedClientRequest` here? Should we use some other type of exception? `UnauthorizedClientRequest` is usually raised when a sender doesn't have enough permissions (for example Role) to perform a request.